openc3/python/openc3/interfaces/stream_interface.py

```diff
@@ -35,9 +35,9 @@ def __init__(self, protocol_type=None, protocol_args=[]):
             self.add_protocol(klass, protocol_args, "PARAMS")
 
     def connect(self):
-        super().connect()
         if self.stream:
             self.stream.connect()
+        super().connect()
```

Comment:
Sending the command in the `connect` method only works if we do the `super()` last.